src/UI/Radio.elm

```diff
@@ -79,6 +80,11 @@ type RadioWidth
     | WidthRelative
 
 
+type Disposition
+    = Vertical
+    | Horizontal
+
+
```

Comment:
Although it's not incorrect I think the word "disposition" is more often used referring to a person's mood. I would go instead with "direction" (like the flex direction), "orientation" or "alignment".